README.md

```diff
@@ -216,6 +216,14 @@ Funded by [MOSS](https://www.mozilla.org/en-US/moss/).
 
 Work of [one of contributors](https://github.com/westnordost/StreetComplete/commits?author=matkoniecz) was sponsored by [NLNet Foundation](https://github.com/ralphtheninja/open-funding#nlnet-foundation), work by the [main author](https://github.com/streetcomplete/StreetComplete/commits?author=westnordost) was funded by [OpenStreetMap foundation](https://blog.openstreetmap.org/2020/07/01/osmf-microgrants-program-congratulations-to-selected-projects/) and [Prototype Fund](https://github.com/streetcomplete/StreetComplete#sponsors).
 
+## Time-bounded Opportunities
+
+### 2024 September 13 - Open Tech Fund Impact and Engagement Lab RFP
+
+OTF is soliciting proposals from service providers, organizations, and individuals interested in providing services to our Impact & Engagement Lab.
```

Comment:
probably it should be in quotes, or "our" replaced with "their" - this lab is not run by ralphtheninja/open-funding people, AFAIK

(and this one is getting into regular contracts a bit, but mostly remains on topic, I think)

Reply:
Good catch. I put it in a blockquote - I had just copied this from the site.